### PR TITLE
ps-8.0-5922

### PR DESCRIPTION
**Pull request by patrickbirch**
Added utility-user to index
Updated 5.7 utility-user doc with 8.0.17-8 version info.
Added limitation

### CODE REVIEW

File: doc/source/management/utility_user.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ Utility user
+==============
+
+In verion 8.0.17-8, |Percona Server| has implemented ability to have a |MySQL| user who has system access to do administrative tasks but limited access to user schema. This feature is especially useful to those operating |MySQL| As A Service. 
+
+This user has a mixed and special scope of abilities and protection:
+
+  * Utility user will not appear in the mysql.user table and can not be modified by any other user, including root.
+
+  * Utility user will increase the result in the `SHOW STATUS LIKE '%Connections'` statement.
```
Comment (dutow) [inline doc/source/management/utility_user.rst]: It's either "results" or you could specify that it's about `Threads_connected` and `Threads_created`. And also the `Connections` counter.

File: doc/source/management/utility_user.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+  * Utility user will not appear in :table:`USER_STATISTICS`, :table:`CLIENT_STATISTICS` or :table:`THREAD_STATISTICS` tables.
```
Comment (dutow) [inline doc/source/management/utility_user.rst]: Or in any performance_schema tables.

File: doc/source/management/utility_user.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+As a result of these requirements, it is strongly recommended that a very unique user name and reasonably specific host be used and that any script or tools test that they are running within the correct user by executing 'SELECT CURRENT_USER()' and comparing the result against the known utility user.
+
+Option :variable:`utility_user_password` specifies the password for the utility user and MUST be specified or the server will shut down gracefully with an error.
+
+ Example: :option:`--utility_user_password` =`Passw0rD`;
+
+Option :variable:`utility_user_schema_access` specifies the name(s) of the schema(s) that the utility user will have access to read write and modify. If a particular schema named here does not exist on start up it will be ignored. If a schema by the name of any of those listed in this option is created after the server is started, the utility user will have full access to it.
+
+ Example: :option:`--utility_user_schema_access` =schema1,schema2,schema3 ;
```
Comment (dutow) [inline doc/source/management/utility_user.rst]: The ; is a typo?

File: doc/source/management/utility_user.rst [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+Option :variable:`utility_user_privileges` allows a comma-separated list of extra access privileges to grant to the utility user.
+
+ Example: :option:`--utility-user-privileges` ="CREATE, DROP, LOCK TABLES"
```
Comment (dutow) [inline doc/source/management/utility_user.rst]: Spaces aren't supported between arguments - this should be `CREATE_DROP,LOCK TABLES`

File: doc/source/management/utility_user.rst [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+.. variable:: utility_user_privileges
+
+   :cli: Yes
+   :conf: utility_user_privileges=<privilege1>,<privilege2>,<privilege3>
+   :scope: Global
+   :dyn: No
+   :vartype: String
+   :default: NULL
+
+This variable can be used to specify a comma-separated list of extra access privileges to grant to the utility user. Supported values for the privileges list are: ``SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, RELOAD, SHUTDOWN, PROCESS,FILE, GRANT, REFERENCES, INDEX, ALTER, SHOW DATABASES, SUPER, CREATE TEMPORARY TABLES, LOCK TABLES,EXECUTE, REPLICATION SLAVE, REPLICATION CLIENT, CREATE VIEW, SHOW VIEW, CREATE ROUTINE, ALTER ROUTINE, CREATE USER, EVENT, TRIGGER, CREATE TABLESPACE``
```
Comment (dutow) [inline doc/source/management/utility_user.rst]: Missing space between PROCESS and FILE, and LOCK TABLES and EXECUTE

**Review by dutow — CHANGES_REQUESTED** [review]
Please also fix these for earlier versions.

**Comment by dutow** [comment]
"Updated 5.7 utility-user doc with 8.0.17-8 version info." - I don't see a PR for 5.7 (and 5.6)?

File: doc/source/management/utility_user.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/management/utility_user.rst]: In the 5.6 version, rewrote the sentence.

File: doc/source/management/utility_user.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/management/utility_user.rst]: added in the 5.6 version.

File: doc/source/management/utility_user.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/management/utility_user.rst]: Should I remove the ";" or delete the space between schema3 ;?

File: doc/source/management/utility_user.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/management/utility_user.rst]: fixed in 5.6 version

File: doc/source/management/utility_user.rst (hunk 5 above)
Comment (patrickbirch) [reply doc/source/management/utility_user.rst]: fixed in 5.6 version

**Comment by patrickbirch** [comment]
Created https://github.com/percona/percona-server/pull/3398 for the 5.6 version.

File: doc/source/management/utility_user.rst (hunk 3 above)
Comment (dutow) [reply doc/source/management/utility_user.rst]: Remove.

**Review by dutow — APPROVED** [review]

**Pull request merged**